lib/mongo/event.rb

```diff
@@ -46,3 +38,12 @@ module Event
     DESCRIPTION_CHANGED = 'description_changed'.freeze
   end
 end
+
```

Comment:
Is there a reason for moving the require statements to the bottom (here and below)?

Reply:
Yes, in general the inner classes should be defined after the outer classes are and once the inner classes start referencing things in the outer classes, or outer classes need to derive from something, the inverted definition order ceases to work.